tests/whatweb_test.py

```diff
@@ -521,3 +522,49 @@ def testWhatWebAgent_withUnsupportedSchema_targetShouldNotBeScanned(
     # assert
     assert len(agent_mock) == 0
     assert subprocess_mock.call_count == 0
+
+
+def testWhatWebAgent_whenIPv4AssetDoesNotReachCIDRLimit_doesNotRaiseValueError(
+    test_agent: whatweb_agent.AgentWhatWeb,
+    mocker: plugin.MockerFixture,
+    scan_message_ipv4_with_mask16: message.Message,
+) -> None:
+    """Test the CIDR Limit in case IPV4 and the Limit is not reached."""
+    mocker.patch(
+        "ostorlab.agent.mixins.agent_persist_mixin.AgentPersistMixin.add_ip_network",
+        return_value=False,
+    )
```

Comment:
Isn't this supported in the `agent_persist_mock` fixture ?

Reply:
Nops,
https://github.com/Ostorlab/ostorlab/blob/dda7a27899dfcb1fef156b8ceb78b6a0a725a697/src/ostorlab/testing/agent.py#L68